pyspecProcScripts/load_data.py

```diff
@@ -346,6 +346,11 @@ def proc_Hahn_echoph(s, fl=None):
     return s
 
 
+def proc_spincore_FID_nutation_v1(s, fl=None):
+    s = proc_spincore_generalproc_v1(s, fl=fl)
```

Comment:
You can just use the dictionary to point straight to the general proc function, since you're not doing anything in addition